odpf/assets/facets/v1beta2/lineage.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```

Comment:
@GrayFlash i am thinking may be we can move all facets and common protos to a single file. thoughts? cc @StewartJingga

Reply:
have removed the facets, and now the common has three proto files, `event`, `lineage`, and `owner`. I think it should be fine to have these three in different files for better separation of functionality.

Reply:
Reasoning i am thinking single file at the same level as asset is so that users in extractors does not have to import two packages. and it makes it little easier for them to use. thoughts?

Reply:
was thinking the same thing, but lineage is now also used in `Dashboard.Chart` model.
So that's why we decided to put it in common folder.

Reply:
@StewartJingga How does that impact? I am thinking of a `common.proto` file at the asset level which holds these protos. So we still have these messages separate but no dedicated package is needed.

Reply:
@ravisuhag  Oh I thought you meant it to be inside asset file alongside `Asset` model.
Then yeah we can have all those in `common.proto`.

Reply:
commited the changes.